esmvaltool/_data_finder.py

```diff
@@ -229,25 +229,29 @@ def get_input_fx_dirname_template(variable, rootpath, drs):
     # Set the drs
     _drs = drs.get(project, 'default')
     input_dir = cfg['fx_dir']
+
+    # Loop through fx_files types
     for fx_ind in range(len(variable['fx_files'])):
+
+        # Need to reassign the mip so we can find sftlf/of
+        # make a copy of variable -> new_variable for this
+        new_variable = dict(variable)
+        if new_variable['fx_files'][fx_ind] == 'sftlf' or \
```

Comment:
Use brackets instead of backslashes as recommended in [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).